src/audio/igo_nr/igo_nr.c

```diff
@@ -29,25 +29,6 @@
 
 #define SOF_IGO_NR_MAX_SIZE 4096		/* Max size for coef data in bytes */
```

Comment:
Please add one patch per type of issues fixed.

Reply:
Sure. Just split it to four commits.